doc/releases/changelog-dev.md

```diff
@@ -40,6 +40,37 @@
   Also added ``qml.pauli.trace_inner_product`` that can handle batches of dense matrices.
   [(#6811)](https://github.com/PennyLaneAI/pennylane/pull/6811)
 
+* Added class `qml.FromBloq` that takes Qualtran Bloqs and translates them into equivalent PennyLane operators.
+  For example, we can now import Bloqs and use them in a way similar to how we use PennyLane templates:
+
+  ```python
+  from qualtran.bloqs.phase_estimation import RectangularWindowState, TextbookQPE
+  from qualtran.bloqs.chemistry.trotter.ising import IsingXUnitary, IsingZZUnitary
+  from qualtran.bloqs.chemistry.trotter.trotterized_unitary import TrotterizedUnitary
+
+  nsites = 5
+  j_zz = 2
+  gamma_x = 0.1
+  dt = 0.01
+  indices = (0, 1, 0)
+  coeffs = (0.5 * gamma_x, j_zz, 0.5 * gamma_x)  
+  zz_bloq = IsingZZUnitary(nsites=nsites, angle=2 * dt * j_zz)
+  x_bloq = IsingXUnitary(nsites=nsites, angle=0.5 * 2 * dt * gamma_x)
+  trott_unitary = TrotterizedUnitary(
+      bloqs=(x_bloq, zz_bloq), indices=indices, coeffs=coeffs, timestep=dt
+  )  
+  textbook_qpe = TextbookQPE(trott_unitary, RectangularWindowState(3))
+
+  dev = qml.device("default.qubit")  
+  @qml.qnode(dev)
+  def circuit():
+      qml.FromBloq(textbook_qpe, wires=list(range(8)))
+      return qml.state()
+  
+  circuit()
+  ```
+  [(#6921)](https://github.com/PennyLaneAI/pennylane/pull/6921)
+
```

Comment:
Not sure about this changelog. @DSGuala should there be just a simple example? Or what's preferred in this case?

Reply:
I think a simpler example could be nice. Maybe a Bloq with one or two arguments that are easy to define in-line?